Bottom: c5c18547ec01e6676f6539fb1f97168a31b4882b
Top:    3818a1c9acdf1453af03761ad09600ef2a68fb8d
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-03 19:27:15 +0000

Refresh of play-with-locked-object

---

diff --git a/src/hid/gtk/gtkhid-gdk.c b/src/hid/gtk/gtkhid-gdk.c
index c488042..c0f5a97 100644
--- a/src/hid/gtk/gtkhid-gdk.c
+++ b/src/hid/gtk/gtkhid-gdk.c
@@ -1389,3 +1389,8 @@ ghid_cancel_lead_user (void)
   priv->lead_user_timer = NULL;
   priv->lead_user = false;
 }
+
+void
+ghid_set_lock_effects (hidGC gc, AnyObjectType *object)
+{
+}
